packages/PackageBuilder/src/DependencyInjection/CompilerPass/AutowireSinglyImplementedCompilerPass.php

```diff
@@ -13,6 +14,18 @@
  */
 final class AutowireSinglyImplementedCompilerPass implements CompilerPassInterface
 {
+    /**
+     * Classes that are definitions, but extend/implement non-existing code
+     * @see https://github.com/Symplify/Symplify/issues/1223
+     * @var string[]
+     */
+    private $excludedPossibleFatalClasses = [
```

Comment:
Maybe this could be `public static` so that people can simply add in another class if they find it without waiting for new symplify release or using composer patches? I know it breaks encapsulation but this is only a workaround anyway - the proper fix is the try-catch.

Reply:
Oh, right, I forgot that the CompilerPass is added manually in the application and not by a bundle. So yeah, ctor is better then.

Btw. you can use `::class`. It does not trigger autoloading. https://3v4l.org/IrFTj

Reply:
But the class has to exits, right?

Reply:
They don't. See the link above.